packages/frontend/mobile/src/components/app-tabs/index.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  WorkbenchLink,
+  WorkbenchService,
+} from '@affine/core/modules/workbench';
+import { AllDocsIcon, SearchIcon } from '@blocksuite/icons/rc';
+import { useLiveData, useService } from '@toeverything/infra';
+
+import { HomeIcon } from './home-icon';
+import * as styles from './styles.css';
+
+interface Route {
+  to: string;
+  Icon: React.FC;
+  LinkComponent?: React.FC;
```

Comment:
never used？

Reply:
Yes, it is not used yet. I'm concerned that there may be routers outside of the workbench being used as main tab, so I added this one.